webcompat/static/css/development/components/search-issue.css

```diff
@@ -68,3 +68,7 @@
     align-items: center;
     margin: 3em 0 0 0;
   }
+
+  .wc-SearchIssue-github .wc-Link { 
```

Comment:
Me personally don't like to add styles to tags. For me, this is a more dangerous side effect then adding a class. 

While `.wc-SearchIssue-github` seems like a closed module used just for the footer information, so adding an already existing class should make it independent?